wheel/py_streamable/src/lib.rs

```diff
@@ -63,8 +63,8 @@ pub fn py_streamable_macro(input: TokenStream) -> TokenStream {
 
                     #[staticmethod]
                     pub fn from_bytes(blob: &[u8]) -> PyResult<Self> {
-                        let mut de = ChiaDeserializer::from_slice(blob)?;
-                        Self::deserialize(&mut de).map_err(|e| e.into())
+                        let mut input = std::io::Cursor::<&[u8]>::new(blob);
+                        Self::parse(&mut input).map_err(|e| <chia::chia_error::Error as Into<PyErr>>::into(e))
```

Comment:
Note that the `?` operator does a `Err(From::from(e))` implicitly, so you might be able to simplify this if you implement `From<chia_error::Error>` for `PyErr`. Maybe this isn't possible, but might be worth looking into.

Reply:
yeah, I'm aware of that (but probably fail to take advantage of it at times). I'm not 100% sure, but I believe the reason I made this change was because a file using this macro probably hadn't imported `chia::chia_error::Error`, so it wasn't in scope. I think it's a bit problematic when the macro requires the user to import a bunch of types so I tried to make the macro generate as many fully qualified types as possible. So I believe fully qualifying the type here makes the macro simpler to use.

I think it's worth looking into simplifying it still, but in a separate PR.